core/src/main/web/app/helpers/helper.js

```diff
@@ -205,6 +205,22 @@
       getBaseUrl: function() {
         return bkUtils.getBaseUrl();
       },
+      openNotebookInNewWindow: function (notebookUri, uriType, readOnly, format) {
+
+        var path = "/open?uri=" + window.encodeURIComponent(notebookUri);
+        if (uriType) {
+          path += "&type=" + uriType;
```

Comment:
No need to do this manually, you can use $httpParamSerializer.

https://docs.angularjs.org/api/ng/service/$httpParamSerializer